src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -165,10 +168,27 @@ def __str__(self):
         return ''.join(self.header) + '\n\n'.join(map(str, self.masks))
 
 
-def get_comment():
+def get_comment(bugs):
     """Spawn editor to get mask comment."""
     tmp = tempfile.NamedTemporaryFile(mode='w')
+    if len(bugs) > 0:
+        # Bug #A, #B, #C
+        tmp.write(f'\nBug {", ".join([f"#{bug}" for bug in bugs])}')
     tmp.write("\n\n# Please enter the mask message. Lines starting with '#' will be ignored.\n")
+    tmp.write("#\n")
+    tmp.write("# - Best last rites (removal) practices -\n")
+    tmp.write("#\n")
+    tmp.write("# Include the following info:\n")
+    tmp.write("# a) reason for masking\n")
+    tmp.write("# b) bug # for the removal (and yes you should have one)\n")
+    tmp.write("# c) date of removal (either the date or \"in x days\")\n")
```

Comment:
I considered omitting this line, as the removal date is added automatically with `-r`

Reply:
Instead of using lots of unnecessary write() calls, a single write() call using textwrap.dedent() should be used. See the tests for an example at https://github.com/pkgcore/pkgdev/blob/main/tests/scripts/test_pkgdev_mask.py#L171.

Reply:
Awesome, thanks! I just went the first implementation here, I'll adjust it.

Reply:
Done.